code/modules/reagents/Chemistry-Machinery.dm

```diff
@@ -575,6 +575,67 @@
 	desc = "Used to create condiments and other cooking supplies."
 	condi = 1
 
+//////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+/obj/machinery/chem_master/constructable
+	name = "ChemMaster 2999"
+	desc = "Used to seperate chemicals and distribute them in a variety of forms."
+	
+/obj/machinery/chem_master/constructable/New()
+	..()
+	component_parts = list()
+	component_parts += new /obj/item/weapon/circuitboard/chem_master(null)
+	component_parts += new /obj/item/weapon/stock_parts/manipulator(null)
+	component_parts += new /obj/item/weapon/stock_parts/console_screen(null)
+	component_parts += new /obj/item/weapon/reagent_containers/glass/beaker(null)
+	component_parts += new /obj/item/weapon/reagent_containers/glass/beaker(null)
+	
+/obj/machinery/chem_master/attackby(var/obj/item/B as obj, var/mob/user as mob, params)
+
+	if(default_deconstruction_screwdriver(user, "mixer0", "mixer0", B))

```

Comment:
This is something I'm aware of but as far as I know there aren't any open sprites for it and really I only have about a 50% idea of what I'm doing here. I can just switch it to mixer0_nopower.